### PR TITLE
Update workflow docs for Harbor 0.5.0 [skip ci]

### PR DESCRIPTION
**Pull request by sflxn**
Update the workflow docs with new information on Harbor 0.5.0.  It
is mostly deletion of content that are no longer necessary with the
0.5.0 release.

### CODE REVIEW

**Comment by anchal-agrawal** [comment]
LGTM

**Comment by chengwang86** [comment]
LGTM

File: doc/application_workflow/harbor_registry/deploy_vch_with_harbor.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## Prerequisite
+
+Harbor requires 60GB or more free space on your datastore.
+
+## Workflow
+
+We will use VIC Engine 0.8.0 and Harbor 0.5.0 for this example.  We will use Ubuntu as OS on our user machine.
+
+If no server certificate and private key are provided during installation, Harbor will self generate these.  It will also provide a self-generated CA (certificate authority) certificate if no server certificate and private key are provided during installation.  The OVA installation guide for Harbor can be found in the [Harbor docs](https://github.com/vmware/harbor/blob/master/docs/installation_guide_ova.md).  Habor requires both an IP address and FQDN (fully qualified domain name) for the the server.  There is also a DHCP install method available for debugging purposes, but it is not a recommended production deployment model.
```
Comment (sflxn) [inline doc/application_workflow/harbor_registry/deploy_vch_with_harbor.md]: fixed.

**Comment by sflxn** [comment]
ping

**Pull request merged**